app/src/main/java/com/safetyheads/akademiaandroida/LoginFragment.kt

```diff
@@ -0,0 +1,18 @@
+package com.safetyheads.akademiaandroida
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+class LoginFragment : Fragment() {
```

Comment:
Could you also include this Fragment in the nav_graph.xml so we can navigate to it?